packages/theme/src/cli/commands/theme/push.ts

```diff
@@ -139,33 +134,45 @@ export default class Push extends ThemeCommand {
 
   async run(): Promise<void> {
     const {flags} = await this.parse(Push)
```

Comment:
The logic on the LHS here has been copied to the service layer as well as included in `execLegacyPush()`

I'm keeping it here in the command layer to service the `CLI2` logic - in this case I think it's cleaner / easier to change in the future if we have some temporary duplication here